docs/source/en/using-diffusers/callback.md

```diff
@@ -19,13 +19,50 @@ The denoising loop of a pipeline can be modified with custom defined functions u
 
 This guide will demonstrate how callbacks work by a few features you can implement with them.
 
+## Official callbacks
+
+We provide a list of callbacks you can plug into an existing pipeline and modify the denoising loop. This is the current list of official callbacks:
+
+- `SDCFGCutoffCallback`: Disables the CFG after a certain number of steps for SD 1.5 pipelines.
+- `SDXLCFGCutoffCallback`: Disables the CFG after a certain number of steps for SDXL pipelines.
+- `IPAdapterScaleCutoffCallback`: Disables the IP Adapter after a certain number of steps.
+
+To set up a callback, you need to specify the number of steps after which the callback comes into effect. You can do so by using either one of these two arguments
+
+- `cutoff_step_ratio`: Float number with the ratio of the steps.
+- `cutoff_step_index`: Integer number with the exact number of the step.
+
```

Comment:
Would be nice to also add a block of text mentioning that these callbacks are used with other compatible pipelines such as ControlNet, MultiControlNet, as well which I believe is the case?